govcd/metadata.go

```diff
@@ -839,3 +839,34 @@ func (orgVdcNetwork *OrgVDCNetwork) DeleteMetadataEntry(key string) error {
 func (orgVdcNetwork *OrgVDCNetwork) DeleteMetadataEntryAsync(key string) (Task, error) {
 	return deleteMetadata(orgVdcNetwork.client, key, strings.ReplaceAll(orgVdcNetwork.OrgVDCNetwork.HREF, "/api/", "/api/admin/"))
 }
+
+// OpenAPI metadata functions
+
+// GetMetadata returns OpenApiOrgVdcNetwork metadata.
+// TODO: This function is currently using XML underneath as metadata is supported in v37.0 and at the moment is in alpha state. See https://github.com/vmware/go-vcloud-director/pull/455
```

Comment:
Is comment(TODO) is correct, if not please correct in other functions also

Reply:
Yep. The idea is that when v37.0 is released we can come back to this and improve these functions with the PR shown in the comment: #455 (basically removes XML in favor of OpenAPI)

Reply:
Improved the message to not to be confusing. Thanks!